tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -113,20 +112,17 @@ - (instancetype)initWithValue:(NSInteger)value {
 
     NSPredicate* unArchivedPredicate = [NSKeyedUnarchiver unarchiveObjectWithData:data];
     ASSERT_TRUE_MSG(unArchivedPredicate != nil, "FAILED: unArchivedPredicate should be non-null!");
-
-    [predicate release];
 }
 
 TEST(NSPredicate, copy) {
-    NSPredicate* predicate = [[NSPredicate alloc] init];
+    NSPredicate* predicate = [NSPredicate predicateWithFormat:@"1<3"];
     ASSERT_TRUE_MSG(predicate != nil, "FAILED: predicate should be non-null!");
 
     NSPredicate* copyObj = [predicate copy];
     ASSERT_TRUE_MSG(copyObj != nil, "FAILED: copyObj should be non-null!");
 
     ASSERT_OBJCEQ_MSG(predicate, copyObj, "FAILED: objects do not match.");
 
-    [copyObj release];
```

Comment:
instead should remove [predicate release];

---
In reply to: [73250756](https://github.com/Microsoft/WinObjC/pull/788#discussion_r73250756) [](ancestors = 73250756)